toolkit/tools/internal/isogenerator/isogenerator.go

```diff
@@ -0,0 +1,338 @@
+// Copyright Microsoft Corporation.
+// Licensed under the MIT License.
+
+package isogenerator
```

Comment:
why is this under internal and not under pkg?

Reply:
`pkg` is for official APIs, particularly those that have an equivalent CLI too. `internal` is for internal implementation details. `internal` made more sense to me for this class.